misc/webrtc-utils/src/stream.rs

```diff
@@ -53,6 +54,8 @@ const VARINT_LEN: usize = 2;
 const PROTO_OVERHEAD: usize = 5;
 /// Maximum length of data, in bytes.
 const MAX_DATA_LEN: usize = MAX_MSG_LEN - VARINT_LEN - PROTO_OVERHEAD;
+/// FIN_ACK timeout
+const FIN_ACK_TIMEOUT: Duration = Duration::from_secs(10);
```

Comment:
Is this number part of specs or was discussed somewhere? 
This is time that we'll have to wait each time when closing a stream if the remote doesn't support `FIN_ACK` right?

Reply:
Yes, that's right. I follow the go-libp2p implementation. https://github.com/libp2p/go-libp2p/blob/master/p2p/transport/webrtc/stream.go#L45